cmd/kar-controllers/app/options/options.go

```diff
@@ -60,6 +68,30 @@ func (s *ServerOption) loadDefaultsFromEnvVars() {
 	if envVarExists && strings.EqualFold(dispatcherMode, "true") {
 		s.Dispatcher = true
 	}
+
+	dynamicpriority, envVarExists := os.LookupEnv("DYNAMICPRIORITY")
+	s.DynamicPriority = false
+	if envVarExists && strings.EqualFold(dynamicpriority, "true") {
+		s.DynamicPriority = true
+	}
+
+	preemption, envVarExists := os.LookupEnv("PREEMPTION")
+	s.Preemption = false
+	if envVarExists && strings.EqualFold(preemption, "true") {
+		s.Preemption = true
+	}
```

Comment:
@zhangliresearch Since the current implementation does not support `DynamicPriority=true` and `Preemption=true` can you update this logic to enforce `Preemption=false` if `DynamicPriority=true`?  Please let me know your thoughts.  Thank you.